infra/modules/network/main.tf

```diff
@@ -0,0 +1,30 @@
+data "aws_availability_zones" "available" {}
+
+locals {
+  vpc_cidr               = "10.0.0.0/20"
+  num_availability_zones = 3
+  availability_zones     = slice(data.aws_availability_zones.available.names, 0, local.num_availability_zones)
+}
+
+module "aws_vpc" {
+  source = "terraform-aws-modules/vpc/aws"
```

Comment:
They used [this one on dmod](https://github.com/navapbc/bid-fed-cms-dmod2-tech-challenge/blob/98fe67f21b26e12a7596fbbe5153c3b7a8e9a21c/infra/modules/vpc/main.tf#L16-L17) and pinned the version, which might be wise. If that doesn't make sense for this repo, we can always pin it where it's used.